AUTHORS

```diff
@@ -13,4 +13,5 @@ Haoran Zhang <andrewzhr9911@gmail.com>
 Hazem Alrawi <hazemalrawi7@gmail.com>
 Shahryar Soltanpour <shahryar.soltanpour@gmail.com>
 Shikhar Soni <shikharish05@gmail.com>
-Nguyen Cong Nhat Le <lenguyencongnhat2001@gmail.com>
\ No newline at end of file
+Nguyen Cong Nhat Le <lenguyencongnhat2001@gmail.com>
+Chao Gu <chadraven369@gmail.com>
```

Comment:
You've added yourself in #212. You may need to revert this or it might cause conflict.